src/util/StringUtils.h

```diff
@@ -665,18 +263,19 @@ inline size_t findLiteralEnd(const std::string_view input,
 }  // namespace ad_utility
 
 // these overloads are missing in the STL
+// they can be constexpr once the compiler completely supports C++20
 inline std::string operator+(const std::string& a, std::string_view b) {
   std::string res;
-  res.resize(a.size() + b.size());
-  std::memcpy(res.data(), a.data(), a.size());
-  std::memcpy(res.data() + a.size(), b.data(), b.size());
+  res.reserve(a.size() + b.size());
+  res += a;
+  res += b;
   return res;
 }
 
 inline std::string operator+(char c, std::string_view b) {
   std::string res;
-  res.resize(1 + b.size());
-  res[0] = c;
-  std::memcpy(res.data() + 1, b.data(), b.size());
+  res.reserve(1 + b.size());
+  res += c;
+  res += b;
```

Comment:
But this could be implemented using absl::StrCat in one line.
(You have to explicitly construct a string_view from `c`, but it's still one line:).

Then you could also use `absl::StrCat` in the above overload.

Reply:
I think the strCat is not needed here, and especially, they also have the logic for  converting
numbers which i think comes at a price.